run-or-die

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 run_or_die() {
-  local cmd="$1"
+  local cmd="$@"
   echo "running: $cmd"
   $cmd || exit $?
```

Comment:
digging into this further, using the local cmd as the command to run is not safe.  If the argument contained spaces and was wrapped in quotes, the quotes would be lost when the command runs.

changing this to

```
run_or_die() {
  echo "$@"
  "$@" || exit $?
}
```